### DISCUSSION TITLE
Esc not working when using SumatraPDF from command line ?

### DISCUSSION THREAD

**Discussion by giwul70** [sumatrapdfreader/sumatrapdf#3256] [Q&A]
I am new to SumatraPDF, please bear with me.
Since yesterday I am using the portable version of SumatraPDF.

Created a button in another application with the following instruction:
"D:\blahblah\SumatraPDF\SumatraPDF-3.4.6-64.exe" -new-window "%1"

This works well.
Regretfully hitting Esc does not close the window.

When using:  -presentation it gets me the full screen, hit Esc, I return to the same window as when launching -new-window.

I wonder whether there is a way to have the Esc key do the same as ctrl-F4 ?

Thanks!

**Comment by GitHubRulesOK** [comment]
yes
but beware there are 2 settings one is depreciated external command line (-esc-to-exit) the preferred means is use the value `EscToExit = true` in the settings file. Sometimes setting it on and not remembering you may later force an unwanted exit behaviour hence default is false.
